vega/static/vega.js

```diff
@@ -6,6 +6,21 @@ var type = "{type}";
 var output_area = this;
 
 require(['nbextensions/jupyter-vega/index'], function(vega) {{
+  var target = document.createElement('div');
+  target.id = {id};
+  target.className = 'vega-embed';
+
+  var style = document.createElement('style');
+  style.textContent = [
+    '.vega-embed .error p {{',
+    '  color: firebrick;',
+    '  font-size: 14px;',
+    '}}',
+  ].join('\\n');
+
+  element[0].appendChild(target);
```

Comment:
What is `element`?

Reply:
It is a jquery wrapped DOM element inside the output area, that IPython guarantees to be there. It is documented [here](http://ipython.readthedocs.io/en/stable/api/generated/IPython.display.html#IPython.display.Javascript.__init__). I will add a reference to the code to make it clear going forward.